### DISCUSSION TITLE
Can't figure out how to make waline appear

### DISCUSSION THREAD

**Discussion by didntchooseaname** [walinejs/waline#2063] [General]
Hello and thank you for your work ! 

I use a static files generator called "retype" based on yml for conf and markdown for content.
I followed the documentation and deployed the waline project with vercel from github, and it worked flawlesly (https://comments-lake-kappa.vercel.app/).

In facts, there is a _includes folder when i can put code in different places (head.html, bottom.md...), that code will be applied on each generated pages.

I use leancloud and set the env variables in vercel accordingly to the documentation.

So i followed the next setps and i include the css between `<head>` :

```html
<link rel="stylesheet" href="https://unpkg.com/@waline/client@v2/dist/waline.css"/>
```

and the js script with the html div in the `body` : 

```js
  <div id="waline">
  <p>test</p>
  </div>
  <script type="module">
    import { init } from 'https://unpkg.com/@waline/client@v2/dist/waline.mjs';

    init({
      el: '#waline',
      serverURL: 'https://comments-lake-kappa.vercel.app/',
      lang: 'en',
      pageview: true,
    });
  </script>
```

I deployed that code to a preview branch of the website : https://docs-contactit-fr-git-preview-didntchooseaname.vercel.app/

The change are effective, the css is included and the html code too, but there is no block on the website at the very bottom of each pages.

The 'test' that i put between `<p>` in the div appear at the bottom.

Had an error while fetch the css ans an uncaught syntax :

<img width="838" alt="screen" src="https://github.com/walinejs/waline/assets/126179594/2af54a31-cde4-4161-8a41-779337b9776b">

I tried on diferent stock browsers.

Do i misunderstood somthing ? Thank you.

**Comment by lizheming** [comment]
serverURL should not include `/` at the end. Please try update serverURL to `https://comments-lake-kappa.vercel.app` and it may works well.

**Reply by didntchooseaname** [reply]
Hello and thank you for your answer. As i tested previously and retested now, delete the slash at the end of the link changed nothing.

**Reply by lizheming** [reply]
It seems your code has been higlighted, so it meets javascript's SyntaxError
![image](https://github.com/walinejs/waline/assets/424491/3f0d0cdc-4169-413d-bda5-43e0fc382833)
